include/darwin/hax_mac.h

```diff
@@ -198,6 +198,8 @@ extern int default_hax_log_level;
             v->panicked = 1;          \
         }
 
+#define hax_assert assert
```

Comment:
For Linux and Windows, `hax_assert` is defined as/like a function, and I think we should follow that style here:

```suggestion
#define hax_assert(condition) assert(condition)
```